aviary/variable_info/variable_meta_data.py

```diff
@@ -1666,7 +1666,7 @@
                      "LEAPS1": None
                      },
     units='unitless',
-    types=(str, Path, None),
```

Comment:
I'm not sure the best way to handle it, but I don't want there to be an actual "default" data file that gets used if the user doesn't specify. There has to be a `default_value` because of the way Aviary is set up so I put None. But None isn't actually a acceptable type for `DATA_FILE`, if in your csv you put `DATA_FILE = None` you should get told that's not valid. An error's going to get triggered somewhere along the way, the question is where that would be the most helpful to the user to occur.